src/telliot_feeds/reporters/reporter_autopay_utils.py

```diff
@@ -122,14 +123,25 @@ async def get_current_feeds(self, require_success: bool = False) -> Any:
                     )
                 )
         multi_call = Multicall(calls=calls, _w3=self.w3, require_success=require_success)
-        data = await multi_call.coroutine()
+        try:
+            data: Dict[str, Any] = await multi_call.coroutine()
+        except ValueError as e:
+            if "unsupported block number" in str(e):
+                msg = f"Unsupported block number in multicall request, KeyError: {e}"
+                logger.warning(msg)
+                return None
+            else:
+                msg = f"Error in multicall request, KeyError {e}"
+                logger.warning(msg)
+                return None
         # remove status boolean thats useless here
         try:
-            data.pop("disregard_boolean")
+            _ = data.pop("disregard_boolean")
+            return data
         except KeyError as e:
             msg = f"No feeds returned by multicall, KeyError: {e}"
```

Comment:
It's a little unclear that this this key not being in the dict means no feeds were returned, maybe add a comment why. I know this is unrelated to the issue, but this is as good an opportunity as any..